Sources/hostmgr/commands/vm/VMStart.swift

```diff
@@ -74,17 +74,18 @@ struct VMStartCommand: AsyncParsableCommand {
 
     var sharedPaths: [LaunchConfiguration.SharedPath] {
```

Comment:
btw, this could be the occasion to better document where those shared paths are mounted in the VM, as this info is pretty obscured deep in the Virtualization framework docs

```suggestion
    // Those paths are auto-mounted in the VM in the `/Volumes/My Shared Files/` path
    // See https://developer.apple.com/documentation/virtualization/vzvirtiofilesystemdeviceconfiguration
    // See Sources/libhostmgr/VM/LaunchConfiguration.swift#L69-L82
    var sharedPaths: [LaunchConfiguration.SharedPath] {
```